statannot/comparisons_corrections/funcs.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+
+from statannot.comparisons_corrections.utils import check_pval_correction_input_values, return_results, \
+    get_num_comparisons
+
+
+def bonferroni(p_values, num_comparisons='auto'):
```

Comment:
I will actually suggest a `stats` subpackage, in another PR, including test functions, comparison corrections functions, StatResults, and their utils, so that they are distinct from the annotation part.

Reply:
To continue on the use of the `ComparisonsCorrection` class, this will come handy to be able to specify more parameters without passing through to `add_annotation` but with first initializing the correction function and then passing it to `add_annotation`.